sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -186,6 +186,9 @@ def __init__(self,
 
       **kwargs: Add overrides for arguments passed in flags.
     """
+    # Initializing logging configuration in case the user did not set it up.
+    logging.basicConfig()
```

Comment:
Pleas only call this if the set of root loggers is empty, preserving the previous behavior. (Maybe put it into an init_logging util in case we discover more places we want it.)

Reply:
Do you mean the set of root logging handlers? `basicConfig` itself is a noop if there are root logging handlers configured. Would it be okay to move forward with this, and have a larger improvement as part of addressing BEAM-3177?

Reply:
Ah, yes, looks like you're right. (I was looking at logging.info() and friends that conditionally call this, but I suppose that's to avoid the lock, not because it isn't idempotent.